### ISSUE TITLE
Check the joint that's passed to setTargetPoseRelative matches the group it's in.

### ISSUE THREAD

**Issue by 130s** [start-jsk/rtmros_hironx#61]
For example, this code "runs", but probably nobody wants to do this:

```
robot.setTargetPoseRelative('larm', 'RARM_JOINT5', dx=0.01, tm=5)
```

(What would happen is the left hand tries to move toward where the right hand is, and hit)

Since I assume there's no way for `hrpsys` to know which `eef` belongs to which `joint group` but [`setTargetPoseRelative` still requires both `group name` and `joint name`](http://code.google.com/p/hrpsys-base/source/browse/trunk/python/hrpsys_config.py?spec=svn1015&r=1014#779), solution needs to be added on robot client side (ie. `HIRONX` class). 

Discussed with @emijah and [his comment](https://github.com/start-jsk/hrpsys/issues/35#issuecomment-37147513):

> After talking with @130s we found that having an argument for the group and another for the end effector name can be misleading. One solution would be to have a dictionary to get end effector names from the group name.
> One way would be to use the last name on the rarm/larm list.

**Comment by k-okada** [comment]
```
def setTargetPoseRelative(self, gname, eename, dx=0, dy=0, dz=0, dr=0, 
dp=0, dw=0, tm=10, wait=True):
```

actually eename is not end-effector name, but that is the frame that to 
calculate Relateive Pose, 

On Mon, Mar 10, 2014 at 10:42 AM, Isaac Isao Saito <notifications@github.com

**Comment by 130s** [comment]
Should I add a change to `hrpsys`.`HrpsysConfigurator` or does this only need to be implemented in `hironx_client`?

**Comment by k-okada** [comment]
basically we try to think about implement in upstream code, so i think it is better to  consider if it is possible to change `hrpsys.HrpsysConfigurator` first

**Comment by 130s** [comment]
> ```
> def setTargetPoseRelative(self, gname, eename, dx=0, dy=0, dz=0, dr=0, dp=0, dw=0, tm=10, wait=True): 
> ```
> 
> actually eename is not end-effector name, but that is the frame that to calculate Relateive Pose,

Thank you, this should be addressed in #87.
